sensor/common/registry/registry_store.go

```diff
@@ -18,15 +20,27 @@ import (
 	"github.com/stackrox/rox/pkg/sync"
 	"github.com/stackrox/rox/pkg/tlscheck"
 	"github.com/stackrox/rox/pkg/urlfmt"
+	"github.com/stackrox/rox/pkg/utils"
 )
 
 const (
 	pullSecretNamePrefix = "PullSec"
 	globalRegNamePrefix  = "Global"
 )
 
+type tlsCheckResult uint8
```

Comment:
I'm thinking it may make sense to put the `tlsCheck` stuff into a separate file

Reply:
https://github.com/stackrox/stackrox/commit/642be4031dd69b7d9eb121b7b0243c43c31b6e71